src/ruleLoader.ts

```diff
@@ -107,16 +107,30 @@ function transformName(name: string): string {
  * @param ruleName - A name of a rule in filename format. ex) "someLintRule"
  */
 function loadRule(directory: string, ruleName: string): RuleConstructor | "not-found" {
-    const fullPath = path.join(directory, ruleName);
-    if (fs.existsSync(`${fullPath}.js`)) {
-        const ruleModule = require(fullPath) as { Rule: RuleConstructor } | undefined;
+    const ruleFullPath = getRuleFullPath(directory, ruleName);
```

Comment:
Before it wasn't very explicit. The variable indicated that it would be the path, but it's just a part of the path that misses the extension.

To make it aligned with the method name I thought it would make sense to be more explicit about it.